### PR TITLE
Added function to generate name in SheetService.

### LINKED ISSUE #243: Add function to sheet service to get name of sheet for timing file
Currently the name of a sheet is determined in different places, which is not really good for maintainability. Add function to sheet service instead.

### PR DESCRIPTION
**Pull request by marijnvanderhorst**
#### What have I done
Fixes #243.
Added function to generate name of a sheet in SheetService. This makes sure we don't need to generate in multiple parts of the program.

#### Testing
See unit tests.

#### Addresses SRD section(s)
Section 3.1.12 has been updated.

### CODE REVIEW

File: src/app/backend/sheet-service/sheet.service.ts [hunk 1]
```diff
@@ -1,6 +1,8 @@
 import { Injectable } from '@angular/core';
 import { SheetState } from './sheet-state/sheet-state';
 import { BehaviorSubject, Observable } from 'rxjs';
+import { SheetTimingState } from './sheet-timing-state/sheet-timing-state';
+import { IllegalArgumentError } from '../../errors/illegal-argument-error';
```
Comment (TijnBertens) [inline src/app/backend/sheet-service/sheet.service.ts]: Unused import

File: src/app/backend/sheet-service/sheet.service.spec.ts [hunk 2]
```diff
@@ -2,6 +2,8 @@ import { TestBed } from '@angular/core/testing';
 
 import { SheetService } from './sheet.service';
 import { SheetState } from './sheet-state/sheet-state';
+import { SheetTimingState } from './sheet-timing-state/sheet-timing-state';
+import { IllegalArgumentError } from '../../errors/illegal-argument-error';
```
Comment (TijnBertens) [inline src/app/backend/sheet-service/sheet.service.spec.ts]: Unused import

Comment (marijnvanderhorst) [reply src/app/backend/sheet-service/sheet.service.spec.ts]: removed

File: src/app/backend/sheet-service/sheet.service.ts (hunk 1 above)
Comment (marijnvanderhorst) [reply src/app/backend/sheet-service/sheet.service.ts]: removed

File: src/app/backend/sheet-service/sheet.service.spec.ts (hunk 2 above)
Comment (marijnvanderhorst) [reply src/app/backend/sheet-service/sheet.service.spec.ts]: removed

**Pull request merged**